pyproject.toml

```diff
@@ -1,10 +1,56 @@
-#[project]
-#requires-python = ">=3.9"
-
 [build-system]
-requires = ["setuptools>=62.4.0"]
+requires = ["setuptools>=66.1.0"]
 build-backend = "setuptools.build_meta"
 
+[project]
+name = "picard"
+description = "Picard is a cross-platform music tagger powered by the MusicBrainz database"
+authors = [{name = "The MusicBrainz Team"}]
+license = {text = "GPL-2.0-or-later"}
+keywords = ["musicbrainz", "tagging", "audio"]
+requires-python = ">=3.9"
+classifiers = [
+    "License :: OSI Approved :: GNU General Public License v2 or later (GPLv2+)",
+    "Development Status :: 5 - Production/Stable",
+    "Environment :: MacOS X",
+    "Environment :: Win32 (MS Windows)",
+    "Environment :: X11 Applications :: Qt",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3 :: Only",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12",
+    "Programming Language :: Python :: 3.13",
+    "Operating System :: MacOS",
+    "Operating System :: Microsoft :: Windows",
+    "Operating System :: POSIX :: Linux",
+    "Topic :: Multimedia :: Sound/Audio",
+    "Topic :: Multimedia :: Sound/Audio :: Analysis",
+    "Intended Audience :: End Users/Desktop",
+]
+dynamic = ["version", "readme", "dependencies"]
+
+[project.urls]
+Homepage = "https://picard.musicbrainz.org/"
+Documentation = "https://picard-docs.musicbrainz.org/"
+Repository = "https://github.com/metabrainz/picard.git"
+Issues = "https://tickets.metabrainz.org/projects/PICARD/issues/"
+Changelog = "https://picard.musicbrainz.org/changelog/"
+
+[tool.setuptools]
+ext-modules = [
+    {name = "picard.util._astrcmp", sources = ["picard/util/_astrcmp.c"]},
+]
+
+[tool.setuptools.dynamic]
+version = {attr = "picard.PICARD_VERSION_STR_SHORT"}
+readme = {file = ["README.md"], content-type = "text/markdown"}
+dependencies = {file = ["requirements.txt"]}
```

Comment:
According to https://stackoverflow.com/questions/76722680/what-is-the-best-way-to-combine-conda-with-standard-python-packaging-tools-e-g we can even add optional dependencies:

```toml
[project]
dynamic = ["version", "readme", "dependencies", "optional-dependencies"]
```

```toml
[tool.setuptools.dynamic.optional-dependencies]
dev = { file = ["requirements-dev.txt"] }
build = { file = ["requirements-build.txt"] }
```

Reply:
Yes, but let's do this once we know more about how we handle packages in the future. Right now we don't benefit from this much :)